index.html

```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
 
```

Comment:
- [ ] Kindly add a reduce the spacing above the `button` element and increase the button beneath it to match the design.

- Your design
<img width="332" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/6b91a8c0-414c-4e77-bf0e-8081a1d6ab5c">

- Figma design
<img width="186" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/b11b1153-5d96-4fe2-8fee-8c93b8a62149">